pkgs/by-name/ce/cedar/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cedar";
+  version = "4.2.2";
+
+  src = fetchFromGitHub {
+    owner = "cedar-policy";
+    repo = "cedar";
+    rev = "v${version}";
+    hash = "sha256-p8idQx3HGO/ikL0pDTPXx5+rD2sRpXANqs/g51BL554=";
+  };
+
+  cargoHash = "sha256-GuXifjQkH8jV9mT3UYU6rUZB1qO5Xl9tvYJW7MjcW0U=";
+
```

```suggestion

  passthru = {
    tests.version = testers.testVersion { package = cedar; };
    updateScript = nix-update-script { };
  };

```

Not required, but I'd recommend adding a basic test and update script to make this a bit more maintainable